package.json

```diff
@@ -4,7 +4,8 @@
   "description": "TRIM API wrapper",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/mocha/bin/mocha specs.js",
```

Comment:
We should move tests out into a tests folder or have a convention for test files that mocha picks up. I have seen projects where the test runner simply looks for a glob such as `**/*.spec.js` which allows your specs to be local to the modules they test.

For now I have just included a single `specs.js` for holding all tests.